### DISCUSSION TITLE
I try to Create Custom Cell With Checkbox but can't click it

### DISCUSSION THREAD

**Discussion by Sathaphornx5113** [revolist/revogrid#580] [Announcements]
first time landing in to page can't click check box rowdrag  need to refresh page to click it  i have 2 version of checkbox still have same problem 

![checkbox](https://github.com/user-attachments/assets/752ff66f-e4c2-4e0b-acbc-99c9311c229f)
![checkbox2](https://github.com/user-attachments/assets/81328e61-4284-4a2e-8e27-b4c86b227780)
![checkbox3](https://github.com/user-attachments/assets/d3e6a08d-d95f-4c3f-ab32-9552f93b3f3e)

**Comment by revolist** [comment]
Hello @Sathaphornx5113 ,

Please check example first for select all https://rv-grid.com/demo/vue/vue.select.html
If this not helps could you please provide me with broken example sandbox it would require a lot of time to figure out what is not working for you based on the screenshots. I'll gladly would try to help you.

Thanks!

**Comment by m2a2x** [comment]
Converting to discussion.
